include/zephyr/zbus/zbus.h

```diff
@@ -56,17 +56,23 @@ struct zbus_channel {
 	 * for accessing the channel.
 	 */
 	struct k_mutex *mutex;
-#if (CONFIG_ZBUS_RUNTIME_OBSERVERS_POOL_SIZE > 0) || defined(__DOXYGEN__)
-	/** Dynamic channel observer list. Represents the channel's observers list, it can be empty
+
+#if defined(CONFIG_ZBUS_RUNTIME_OBSERVERS) || defined(__DOXYGEN__)
```

Comment:
Your question is valid. Makes sense. There are some cases it can be used. It is hard to get input from the community. No answers on the zbus discord channel. Hard to know if it would affect some users’ use cases. It would be easy to remove 😬

Reply:
I am going to remove that in this pull request. Ok?